### ISSUE TITLE
Bug: meta title and meta description are not updated for 'subscribe'

### ISSUE THREAD

**Issue by cobbspur** [TryGhost/Ghost#8597]
If you change a post slug to `subscribe` and add meta data, this is not reflected on the page.

This occurs even when subscribers is not activated in labs based off of a regex match in https://github.com/TryGhost/Ghost/blob/02c7d039d80a9cae210d922eceee2588b7620337/core/server/controllers/frontend/context.js#L60 for the subscribe url. 

Steps to Reproduce
1. Make sure subscribers is not enabled in labs
2. Create a new post with the slug `subscribe`
3. Add a meta title and meta description for the post and publish the post.
4. View the page source for the post - note the missing meta data
5. Edit post slug to subscribe2 etc, update post and view page source

Ghost version
master@bc301463c
lts@6e6c8ce66

**Comment by AileenCGN** [comment]
@cobbspur Hmmm.... The question that I'm asking myself is, what should the expected behaviour be, if the subscribers feature **is** enabled, and there's a post slug called `subscribe`. In this case, the subscribe feature route would be enabled and the post becomes inaccessible.

**Comment by kirrg001** [comment]
Can't we simply add `subscribe` to the reserved slugs? Subscribers feature won't be part of the labs forever. cc @ErisDS

**Comment by AileenCGN** [comment]
That's what I was thinking as well. But I reckon some slugs exist already like this? What would happen to those?

**Comment by ErisDS** [comment]
> What should the expected behaviour be, if the subscribers feature is enabled, and there's a post slug called subscribe. In this case, the subscribe feature route would be enabled and the post becomes inaccessible.

> Can't we simply add `subscribe` to the reserved slugs?

When we implemented subscribe, I played with this and thought about it quite a lot.

It is not really possible to add reserved slugs now, because people could have already created posts or pages with these slugs. This highlights to me that reserved slugs are a flawed concept. This feeds into other issues we've had over the years where people wanted to legitimately use certain slugs for purposes E.g. we have `archive` as a reserved slug, yet we don't have any plan to build an archive into Ghost at the moment (we originally did, but dropped it because there is no clear single concept of what an archive should be). So all that we are doing is preventing people from creating their own page, which is odd.

TLDR; I think the reserved slugs needs to be reduced down to the absolute must-haves, and then we should implement behaviour such that one thing overrides another in a sensible way:

- if a user has a `/subscribe/` page and the subscribe feature is not enabled, then that page should work as per usual
- if a user has a `/subscribe/` page and the subscribe feature IS enabled, then the subscribe feature should override the user's own page.

Basically, if Ghost provides a feature, that feature should work, regardless of if a user creates a post or page with a conflicting slug. Although we won't provide errors, it should be easy to figure out what is happening and why because the user will see the Ghost-provided page instead of their  own.

Long term, rather than reserved slugs, I think we will have to implement a centralised concept of "urls" that are in use on a blog - so that we can implement autocomplete, de-duplication and prevent clashes. Don't need that yet though :)
